sdk/python/Makefile

```diff
@@ -54,6 +54,16 @@ lint:: ensure
 	. venv/*/activate && \
 		python -m pyright
 
+lint_fix:: ensure
+	. venv/*/activate && \
+		python -m black $(BLACK_FLAGS)
+	. venv/*/activate && \
+		MYPYPATH=./stubs python -m mypy ./lib/pulumi --config-file=mypy.ini
+	. venv/*/activate && \
+		python -m pylint ./lib/pulumi --rcfile=.pylintrc
+	. venv/*/activate && \
+		python -m pyright
```

Comment:
Do these not have flags to write back? Looks the same as just running the format and lint targets.

Reply:
I haven't found an option for that.  

We could have the `lint_fix` target here just depend on `format` and `lint`, but afaik there's no way to guarantee the order the targets are run in, and often it's useful to run the formatting first as the linters might check that.  So I'd rather keep the duplication, but could be convinced otherwise.

Reply:
But if these don't write back can't this just run format?

Reply:
The intention here is to still get the lint results to the user as well, so they can still fix it if it's necessary.  I guess we could just run `make format` followed by `make lint` here.